internal/textutil/almost_equal.go

```diff
@@ -0,0 +1,67 @@
+package textutil
```

Comment:
can we come up with a better package name? :) how about `textdistance` or `misspell`?

Reply:
The `textutil` is pre-existing, only containing `Diff`. It felt like a logical place to put this. I'm fine putting it in a package on its own, just making sure you didn't miss that.

Reply:
Huh, I totally missed that. The last use of textutil went away when we made packages like `modfile` simply forward to `x/mod`, so the package is now unused and IMO should be deleted. Please pretend it's not there; I will delete it soon.